src/include/stir/VectorWithOffset.h

```diff
@@ -207,6 +207,8 @@ class VectorWithOffset
       \todo in principle reallocation could be avoided when the new range would fit in the
       old one by shifting.
     */
+  // inline virtual void resize(const int min_index, const int max_index, bool initialise_with_0 = true);
+  inline virtual void resize(const int min_index, const int max_index, bool initialise_with_0);
   inline virtual void resize(const int min_index, const int max_index);
```

Comment:
will need a brief doxygen comment, saying it effectively set `initialise_with_0=true`